<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TubeToBili</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>290</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>视频信息</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>110</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>YouTube视频地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>80</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;a href=&quot;https://www.bilibili.com/read/cv18327205&quot;&gt;分区ID列表&lt;/a&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="url">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>110</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Bilibili分区ID：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>290</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string>Bilibili Cookie</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>110</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>SESSDATA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SESSDATA">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bili_jct">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>120</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>bili_jct</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DedeUserID__ckMd5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>120</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>DedeUserID__ckMd5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>120</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>DedeUserID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DedeUserID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="access_token">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>120</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>access_token</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="refresh_token">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>120</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh_token</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>开始转载</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>230</width>
     <height>410</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Made By Gingmzmzx With Love</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
